src/containers/ClassifierContainer.jsx

```diff
@@ -168,11 +169,24 @@ class ClassifierContainer extends React.Component {
       </main>
     );
   }
+  
+  //----------------------------------------------------------------
+  
+  componentWillReceiveProps(next) {
+    if (!this.props.selectedAnnotation && next.selectedAnnotation) {
+      this.setState({
+        popup: <SelectedAnnotation annotation={next.selectedAnnotation} />
+      });
+    }
+  }
 
   //----------------------------------------------------------------
 
   closePopup() {
     this.setState({ popup: null });
```

Comment:
I thought I was repeating some of the same functionality that was being passed to the popup component, but we can keep it abstracted out into individual components and move it later if we think it's necessary. Popups are also used when viewing subject metadata and adding an item to a collection.